build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -126,6 +126,10 @@ trafficEncapMode: {{ .Values.trafficEncapMode | quote }}
 # performs SNAT and this option will be ignored; for other modes it must be set to false.
 noSNAT: {{ .Values.noSNAT }}
 
+# Fully randomize source port mapping in SNAT rules enforced by Nodes for egress traffic from local
+# Pods to the external network. This has no impact on the SNAT rules for the Egress feature.
```

Comment:
While considering the use cases, I couldn't think of a scenario where users prefer random source port for Node SNAT but prefer original source port for Egress SNAT, or vice verse. And requiring users to set two options of the same name could be error-prone. We received a few false bug reports in the past, which turned out to be because user set a feature gate in controller.conf but not agent.conf. I feel this could be similar, though the consequence of any values of the two options is not really critical.

Reply:
I tend to agree with you. That was my preference, but I based the implementation on the user's comment: https://github.com/antrea-io/antrea/issues/6544#issuecomment-2249318867

I think that overall, there is very little reason _not_ to use `--random-fully` if it is available.
There was one other approach I was considering: we can make `egress.snatFullyRandomPorts` a pointer and when omitted it will use whichever value is configured for `snatFullyRandomPorts`. This way we still have the toggle, but it's out of the way, and users only need to set one configuration. I can push a commit so you can take a look at what I mean.

Reply:
@tnqn I pushed an update, PTAL
If you still think that it is not needed and that we should not introduce the complexity in Antrea code, let me know. I don't feel strongly about it and given the lack of tangible use case, I am ok with removing the extra configuration parameter altogether.

Reply:
I still prefer a single configuration parameter a bit for simplicity. Perhaps the new code can be added when there is such a use case.

Reply:
We got good feedback from @Scoobed (who requested the feature) for the given configuration.
"Hiding" the second parameter from the manifest as is the case in the current version may be a good compromise and help avoid confusion? That being said, I am still on the fence myself.

@Scoobed @robbo10 do you really see a need for `egress.snatFullyRandomPorts` and the following installation commands:
```
helm install -n kube-system antrea antrea/antrea --set snatFullyRandomPorts=true --set egress.snatFullyRandomPorts=false
helm install -n kube-system antrea antrea/antrea --set egress.snatFullyRandomPorts=true
```
These commands would enable fine-grained control over which specific SNAT rules use `--random-fully`.

@tnqn and I believe that most users would end up just doing the following, which would set `--random-fully` for all SNAT rules (Node-local and Egress):
```
helm install -n kube-system antrea antrea/antrea --set snatFullyRandomPorts=true
```